packages/errors/src/index.ts

```diff
@@ -59,8 +55,8 @@ export const isWorkflowError = (error: Error) => error.name.includes('WorkflowEr
 /**
  * Returns true if the error is a DynamoDB conditional check exception.
  */
-export const isConditionalCheckException = (error: ErrorWithName) =>
-  error.name === 'ConditionalCheckFailedException';
```

Comment:
We're also checking for a `ThrottlingException` here? Should these both be `ConditionalCheckFailedException`?